### DISCUSSION TITLE
Energy dashboard with single inverter

### DISCUSSION THREAD

**Discussion by azalin1975** [WillCodeForCats/solaredge-modbus-multi#257] [Q&A]
Currently I have added with some success my SolarEdge PV installation (SE5000H inverter) through the modbus over TCP integration.
I though I had succesfully added the device to the energy dashboard but the energy dashboard remains at 0kWh. I have added a card to my regular dashboard that shows actual production. The energy dashboard does not. I noticed that the sensor used to determine the production in the energy dashboard differs form what I used to get the current (haha) value into my dashboard card.
I read a little bit about templates, but can't really make heads or tails of it at the moment.
I have to admit that I am fairly new to HA. So I am still trying to learn to get grips on things.

Things I wonder and hope someone has answers to or help me out with are:
- Why aren't the other sensors not being picked up?
- Why are they not available for selection in the energy dashboard? Currently the sensor SolarEdge AC Energy kWh is used, wouldn't I need to use SolarEdge AC Power instead?
- Do I need to maybe create helpers or fabricate some additional devices?
- Or is this something that needs to be addressed by the developer?

I sincerely hope someone can help me out. Thanks in advance.

On a sidenote: I do see that SolarEdge AC Energy kWh contains data. The energy dashboard just does not show it. It "might" be that I need to be more patient and wait until it get's updated. This could also be an answer to my puzzlement ...

**Comment by WillCodeForCats** [comment]
You will use kilowatt-hour sensors for the Energy dashboard.

Kilowatts is instantaneous power at that moment.

**Reply by azalin1975** [reply]
It all came down to: be patient my young paduwan.
There was just one sensor to select, and after a couple of hours of waiting the dashboard was filled with the new data from the inverter. I had not realised it would take stats by the hour even though the graph says this is exactly what it is going to do.
I just now also added a reader to monitor the P1 port ...

TLDR; I should have waited longer ... ;-) Sorry for all these wasted CPU cycles.
